usagov/vote-gov-drupal
# Vote-739: Config set up for hero section

## Jira ticket

[Vote-739](https://cm-jira.usa.gov/browse/VOTE-739)

## Description
This Pr will set up the config for the hero section of the landing page. 
## Deployment and testing

### Pre-deploy

1. Pull down branch and run `lando retune` 

### Post-deploy

1. Insert any required steps to take before beginning test.

### QA/Test

1. Log into the site and edit the landing/home page -> the expected result is that all text along with image should render 
2. The button for the link will be separate work that can be reused 

## Checklist for the Developer




## Checklist for the Peer Reviewers
